ui/models/job.js

```diff
@@ -115,37 +116,43 @@ export default class JobModel {
     return JobModel.getList(repoName, options, config);
   }
 
-  static async retrigger(jobIds, repoName, getGeckoDecisionTaskId, notify) {
-    const jobTerm = jobIds.length > 1 ? 'jobs' : 'job';
+  static async retrigger(jobs, repoName, notify) {
+    const jobTerm = jobs.length > 1 ? 'jobs' : 'job';
 
     try {
       notify(`Attempting to retrigger/add ${jobTerm} via actions.json`, 'info');
 
-      const jobs = await JobModel.getList(repoName, { id__in: jobIds.join() });
+      const pushIds = [...new Set(jobs.map(job => job.push_id))];
+      const resp = await PushModel.getDecisionTaskMap(pushIds);
```

Comment:
You could add the fetch wrapper `getData` to this method (`getList` is now using it) so you can process error messages.

Reply:
Yeah, you're totally right.  Thanks for catching that. :)  Fixed...